applications/asset_tracker_v2/Kconfig

```diff
@@ -19,6 +19,14 @@ config ASSET_TRACKER_V2_APP_VERSION_MAX_LEN
 	int "Maximum length of the application firmware version"
 	default 150
 
+config ASSET_TRACKER_V2_LTO
```

Comment:
Other libraries/applications may also enable LTO. Should it be mentioned that LTO during linking is applied to the entirety of the files compiled with LTO?

Reply:
Thanks for the comment @jostn  . Do you mean something like this 

> If object files containing GIMPLE bytecode are stored in a library archive, say libfoo.a, it is possible to extract and use them in an LTO link if you are using a linker with plugin support. To create static libraries suitable for LTO, use gcc-ar and gcc-ranlib instead of ar and ranlib; to show the symbols of object files with GIMPLE bytecode, use gcc-nm. Those commands require that ar, ranlib and nm have been compiled with plugin support. At link time, use the flag -fuse-linker-plugin to ensure that the library participates in the LTO optimization process:

from https://gcc.gnu.org/onlinedocs/gcc/Optimize-Options.html  ? If so, I would leave the compiler documentation help developers with this additional info. Perhaps pointing them to this compiler doc may help? But I am afraid that the web links tend to get broken. So I prefer that we leave it to (advanced developers) to find this out themselves.